.github/workflows/release-upload.yml

```diff
@@ -176,6 +176,14 @@ jobs:
           nix build --builders "" --max-jobs 0 ${{ env.LOCKED_URL }}#$derivation
           tree result
           case ${{ matrix.arch }} in
+            "x86_64-darwin" | "aarch64-darwin")
+              NONINTERACTIVE=1 brew install dylibbundler
+              mkdir cardano-cli-${{ matrix.arch }}
+              cp result/bin/cardano-cli cardano-cli-${{ matrix.arch }}/
+              cd cardano-cli-${{ matrix.arch }}
+              dylibbundler -of -cd -b -x cardano-cli -p ./libs -d ./libs
```

Comment:
I think that should go to nix code instead. The point of this job is just take nix artifacts as they are.